bench/chromium/test_cpuprobe.py

```
@@ -0,0 +1,175 @@
+                      "being tracked never reached the record")
+
+
+class EmptySummary(unittest.TestCase):
+    def test_a_run_with_no_usable_figure_reports_instead_of_crashing(self):
+        """percentile() returns None for an empty list, `:.2f` raises TypeError
+        on it, and max([]) raises ValueError. A process that exits before its
+        second sample leaves cores_max None, so a run of only such processes
+        crashed on the way out -- discarding the summary and, worse, hiding the
+        real finding: the probe attached too late to measure anything.
+        """
+        out = os.path.join(os.environ.get("TMPDIR", "/tmp"),
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)